page-blueprints/weather-dashboard-openweathermap/blueprint.yaml

```diff
@@ -1,13 +1,12 @@
 blueprint:
   name: Weather dashboard page Blueprint
   description: An weather dashboard page by Robert Wilson based on orginial page by Leon van der Linden with upgrades from Nate Harris
-  version: "3.2"
+  version: "4.0"
   type: page
   custom_cards:
```

Comment:
This was already in the existing blueprint I believe, but ha-card-weather-conditions is used but not listed here